### PR TITLE
functions: fix route timeout

### PR DESCRIPTION
**Pull request by ccirello**
Fix https://github.com/iron-io/functions/issues/339

### CODE REVIEW

**Comment by denismakogon** [comment]
So, here the thing, we need to have persisted models definitions out of code in order to have good enough user experience during upgrades from version X to version Y.

Another question, if i would create a route before this patch and then will upgrade `functions` API up to this patch how this code will handle cases where model wouldn't have `Timeout` attribute.

Along with that, this code doesn't actually validate if `Timeout` fits `integer` range.

Mentions: https://open-iron.slack.com/archives/functions/p1480362602001126

**Comment by ccirello** [comment]
> So, here the thing, we need to have persisted models definitions out of code in order to have good enough user experience during upgrades from version X to version Y.
> Another question, if i would create a route before this patch and then will upgrade functions API up to this patch how this code will handle cases where model wouldn't have Timeout attribute.

Ideally, we shall have stable model definition by v1, and from there backward compatible changes only. This has been discussed yet. We are still in the Alpha cycle where we all expect things to break often.

Currently, routes whose configuration has `Timeout` zero, will no longer have any timeout. Coding something around expected timeouts is a big mistake. Thus, it should impact anyone doing it right - and it will create a minor nuisance for those using it.

> Along with that, this code doesn't actually validate if Timeout fits integer range.

Indeed it doesn't. But also this PR is WIP. I shall align integer sizes once I regenerated client libraries.

File: fn/routes.go [hunk 1]
```diff
@@ -74,6 +75,11 @@ func routes() cli.Command {
 						Usage: "maximum concurrency for hot container",
 						Value: 1,
 					},
+					cli.DurationFlag{
+						Name:  "timeout",
+						Usage: "route timeout",
+						Value: 30 * time.Second,
```
Comment (pedronasser) [inline fn/routes.go]: 60 or 30 seconds?

Comment (ccirello) [inline fn/routes.go]: done

**Pull request merged**